diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5cfef6b..840666e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,13 +50,13 @@ set(CONFORMANCE_SUFFIX "" )
 #build driver as a dependency of the conformance tests, or other such CMake customization
 include(CMakeVendor.txt OPTIONAL)
 
-# CL_INCLUDE_DIR - path to dir with OpenCL headers
-if(CL_INCLUDE_DIR AND CL_LIB_DIR)
-    link_directories(${CL_LIB_DIR})
-else(CL_INCLUDE_DIR AND CL_LIB_DIR)
-    message(STATUS "OpenCL hasn't been found!")
-    message(FATAL_ERROR "Either install OpenCL or pass -DCL_INCLUDE_DIR and -DCL_LIB_DIR")
-endif(CL_INCLUDE_DIR AND CL_LIB_DIR)
+# # CL_INCLUDE_DIR - path to dir with OpenCL headers
+# if(CL_INCLUDE_DIR AND CL_LIB_DIR)
+#     link_directories(${CL_LIB_DIR})
+# else(CL_INCLUDE_DIR AND CL_LIB_DIR)
+#     message(STATUS "OpenCL hasn't been found!")
+#     message(FATAL_ERROR "Either install OpenCL or pass -DCL_INCLUDE_DIR and -DCL_LIB_DIR")
+# endif(CL_INCLUDE_DIR AND CL_LIB_DIR)
 
 # CLConform_GL_LIBRARIES_DIR - path to OpenGL libraries
 if(GL_IS_SUPPORTED AND CLConform_GL_LIBRARIES_DIR)
@@ -163,6 +163,7 @@ if(WIN32 AND ${CLConform_TARGET_ARCH} STREQUAL "x86")
 endif()
 
 list(APPEND CLConform_LIBRARIES ${OPENCL_LIBRARIES})
+list(APPEND CLConform_LIBRARIES OpenCL::OpenCL)
 if(ANDROID)
     list(APPEND CLConform_LIBRARIES m)
 endif()
